tests/unit/plotting/figure_utils_test.py

```diff
@@ -156,3 +158,67 @@ def test_prepare_figure_external_ax_figsize_none_no_warning():
         assert own is False
     finally:
         plt.close(fig)
+
+
+def test_set_screen_axes_valid():
+    fig, ax = plt.subplots()
+    screen = Screen(width_px=1280, height_px=720, origin='upper left')
+    _set_screen_axes(ax, screen, func_name='dummyplot')
+    assert ax.get_xlim() == (0, 1280)
+    assert ax.get_ylim() == (720, 0)
+    plt.close(fig)
+
+
+@pytest.mark.parametrize('origin', ['lower left', 'center'])
+def test_set_screen_axes_invalid_origin(origin):
+    fig, ax = plt.subplots()
+    screen = Screen(width_px=800, height_px=600, origin=origin)
+    with pytest.raises(ValueError, match='screen origin must be "upper left"'):
+        _set_screen_axes(ax, screen, func_name='dummyplot')
+    plt.close(fig)
+
+
+class DummyScreen(Screen):
```

Comment:
I don't really understand why this class is necessary. Couldn't you just use `Screen` in the tests below?

Reply:
Ah I see now further down below there's the test for negative values. You're right that the check in `_set_screen_axes()` for negative screen values isn't really necessary then, as we can safely assume that the `Screen` is always valid.

Anyway, always good to think about the edge cases. :)